src/WPF/WPF.Viewer/Samples/LocalServer/LocalServerServices/LocalServerServices.xaml

```diff
@@ -6,29 +6,56 @@
             <RowDefinition Height="auto" />
             <RowDefinition Height="auto" />
             <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
             <RowDefinition Height="auto" />
             <RowDefinition Height="auto" />
         </Grid.RowDefinitions>
         <Grid.ColumnDefinitions>
             <ColumnDefinition Width="*" />
             <ColumnDefinition Width="*" />
         </Grid.ColumnDefinitions>
+        <TextBlock Grid.Row="0"
+                   Margin="5"
+                   Text="Set Local Server data path:" />
+        <TextBox x:Name="tempPathBox"
+                 Grid.Row="1"
+                 Grid.ColumnSpan="2"
+                 Margin="5" />
+        <Button x:Name="setTempPathButton"
+                Grid.Row="2"
+                Grid.Column="0"
+                Margin="5"
+                Click="setTempPathButton_Click"
+                Content="Set" />
+        <Button x:Name="defaultTempPathButton"
+                Grid.Row="2"
+                Grid.Column="1"
+                Margin="5"
+                Click="defaultTempPathButton_Click"
+                Content="Default" />
+        <TextBlock x:Name="currentPath"
+                   Grid.Row="3"
+                   Margin="5"
+                   Text="Current path: " />
         <Button x:Name="LocalServerStartButton"
-                Grid.Row="0"
+                Grid.Row="4"
                 Grid.Column="0"
                 Margin="5"
                 Click="StartServerButtonClicked"
                 Content="Start Local Server" />
```

Comment:
Should we disable this button upon starting the sample, until the current path has been set to default or specified by user?